dask-gateway-server/dask_gateway_server/backends/kubernetes/controller.py

```diff
@@ -1052,7 +1052,7 @@ def get_worker_command(self, namespace, cluster_name, config):
             "--name",
             "$(DASK_GATEWAY_WORKER_NAME)",
             "--nthreads",
-            str(int(config.worker_cores_limit)),
+            str(int(config.worker_threads)),
```

Comment:
I think worker_threads is of an `Integer` type already ([traitlets](https://traitlets.readthedocs.io/en/stable/)), while worker_cores_limit was a `Float` traitlet type and needed the cast to integer first.

```suggestion
            str(config.worker_threads),
```